src/redist/redist.csproj

```diff
@@ -21,6 +21,8 @@
     <PackageReference Include="NuGet.Build.Tasks" Version="$(CLI_NuGet_Version)" />
     <PackageReference Include="Microsoft.TestPlatform.CLI" Version="$(CLI_TestPlatform_Version)" />
     <PackageReference Include="Microsoft.TestPlatform.Build" Version="$(CLI_TestPlatform_Version)" />
+    <PackageReference Include="NuGet.Packaging.Core.Types" Version="4.3.0-preview1-2500" />
```

Comment:
Why do we need these here? And why are they not versioned with the NuGet version coming from DependencyVersions.props.